data/korath/korath ships.txt

```diff
@@ -339,7 +339,7 @@ ship "Korath Ravager"
 	explode "huge explosion" 15
 	"final explode" "final explosion large"
 	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed wholesale at the largest Korath facilities, something that, until recently, was generally reserved for the likes of the massive Ik'atilajej and the Korsmanath A'awoj."
-	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances so that it is able to burn enemies in close quarters as well."
+	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances allowing to burn enemies in close quarters."
```

Comment:
```suggestion
	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances allowing it to burn enemies in close quarters."
```
It could also be something like `allowing burning enemies` if you prefer that.

Reply:
That sounds a bit odd to me. I think we're fine with the "allowing it to burn".